src/components/instruments/instruments.stories.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import "./../../styles.scss";
+import { Instruments } from "./instruments";
+import { InstrumentsItem } from "../instruments-item/instruments-item";
+
+export default { title: "Instruments" };
+
+const showMessage = () => alert("Hello, friend.");
+
+export const grid = () => (
+  <Instruments layout="grid">
+    <section>
+      <InstrumentsItem
+        color="#05D6C7"
```

Comment:
Since there can only be 3 color variations and sending hex codes could be error prone, you should set the prop color to be an enum that can receive "red", "pink" or "green". Then you can map each of this to its relevant color.